src/index.spec.js

```diff
@@ -206,6 +210,13 @@ describe('provideState', () => {
       }).toThrowError(new CircularComputedError('circularComputed'))
     })
 
+    it('does not trigger a render when a computed throws', () => {
```

Comment:
I don't believe this is what you're testing here.

Reply:
What you need to test:
- a computed that throws on the first call is visible as `undefined` in the state
- a computed that returns a value and then throws is visible as the previous value